modules/platform/nuxeo-automation/nuxeo-automation-features/src/main/java/org/nuxeo/ecm/automation/core/operations/services/FileManagerImport.java

```diff
@@ -63,8 +63,8 @@
     @Context
     protected OperationContext context;
 
-    @Param(name = "overwite", required = false)
-    protected Boolean overwite = false;
+    @Param(name = "overwrite", alias = "overwite", required = false)
+    protected Boolean overwrite = false;
```

Comment:
In fact, I'm wondering if we should not keep the deprecated field, along with using an alias, just to make sure someone possibly extending this class won't have its compilation broken.
@troger WDYT?